apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/content/subsection/SubSectionGrouper.java

```diff
@@ -46,6 +47,7 @@ public void group(Long pageId, GroupSubSectionRequest request, Long userId) {
       throw new PageNotFoundException(pageId);
     }
     page.groupSubSection(asCommand(userId, request), questionManagementUtil.getQuestionNbsUiComponentUids());
+    entityManager.flush();
```

Comment:
Question: Is the `.flush()` necessary? I thought that if the class was `@Transactional` when the method completed any outstanding changes by the `entityManager` would then be committed.